src/engraving/playback/metaparsers/internal/symbolsmetaparser.cpp

```diff
@@ -286,6 +286,7 @@ void SymbolsMetaParser::doParse(const Ms::EngravingItem* item, const RenderingCo
         types.emplace(mpe::ArticulationType::UpMordent);
         break;
     case Ms::SymId::ornamentMordent:
+    case Ms::SymId::ornamentPinceCouperin:
```

Comment:
Trying to port this PR to 3.x (for inclusion in PR #9000) and wonder where this change needs to go (or wther at all)?

Reply:
Also unsure where `Ms::SymId::ornamentTremplementCouperin` would belong.

Reply:
`Ms::SymId::ornamentTreblementCouperin` was there already
https://github.com/musescore/MuseScore/blob/b8de8be08d5eeb0e660e73eb41edd6f10a9c1152/src/engraving/playback/metaparsers/internal/symbolsmetaparser.cpp#L313

Reply:
Ah, explains my 2nd question. Not the forst nor the last though

Reply:
3. You did
https://github.com/musescore/MuseScore/pull/10282/files#diff-8b5549a030b47fea7e9325a0e5c467a946b66db0992545010dadfb8257521aa4

Reply:
Ah, last quesion is answered as well: I added them already.
Remains the first?

Reply:
I don't think this applies to MU3, since it is part of the new MPE playback system.

Reply:
Yep, just found that out myself seconds ago, thanks

Reply:
Probably answers the 1st question as : no need for this in 3.x